internal/controller/http/router.go

```diff
@@ -27,6 +27,11 @@ import (
 //go:embed all:ui
 var content embed.FS
 
+const (
```

Comment:
I'm not sure about this change it made.  I'll revert tomorrow since it's in the common router.go file.